### PR TITLE
More tests

### PR DESCRIPTION
**Pull request by sunfishcode**

### CODE REVIEW

File: test/core/names.wast [hunk 1]
```diff
@@ -18,52 +18,152 @@
...
+(assert_return (invoke "⁪⁫⁬⁭⁮⁯") (i32.const 31))
+(assert_return (invoke "⁡⁢⁣⁤") (i32.const 32))
+(assert_return (invoke "𐀀󟿿􏿿") (i32.const 33))
+(assert_return (invoke "Z̴͇̫̥̪͓͈͔͎̗̞̺̯̱̞̙̱̜̖̠̏͆̆͛͌͘͞ḁ̶̰̳̭͙̲̱̹̝͎̼͗ͨ̎̄̆͗̿̀́͟͡l̶̷͉̩̹̫̝͖̙̲̼͇͚͍̮͎̥̞̈́͊͗ͦ̈́ͫ̇́̚ͅͅg̶͕͔͚̩̓̐̅ͮ̔̐̎̂̏̾͊̍͋͊ͧ́̆ͦ͞o̡͋̔͐ͪͩ͏̢̧̫̙̤̮͖͙͓̺̜̩̼̘̠́") (i32.const 34))
+(assert_return (invoke "ᅟᅠㅤﾠ") (i32.const 35))
+(assert_return (invoke "︀") (i32.const 36))
+(assert_return (invoke "︄") (i32.const 37))
+(assert_return (invoke "󠄀") (i32.const 38))
+(assert_return (invoke "󠇯") (i32.const 39))
+(assert_return (invoke "̈") (i32.const 40))
```
Comment (jfbastien) [inline test/core/names.wast]: Is there a test with the replacement character  U+FFFD �?

Comment (sunfishcode) [inline test/core/names.wast]: Yes, there's one under "Test the Unicode Specials."

Comment (sunfishcode) [inline test/core/names.wast]: As a bonus, I've now added a few more tests, including some more interesting uses of the replacement character.

**Comment by sunfishcode** [comment]
I've now respun the patches here, to pull out the misc floating-point tests into their own PR (#479), so that this PR just contains names.wast tests, and to make a few other organizational cleanups.

**Comment by sunfishcode** [comment]
Thanks! Perhaps \u escapes are a good opportunity for others interested in writing tests.

**Pull request merged**